### DISCUSSION TITLE
Multi-threaded

### DISCUSSION THREAD

**Discussion by salmanamin0** [framesurge/perseus#162] [Q&A]
**Does Perseus support to create multi-threaded front-end web apps like Yew?**

[Yew.rs](https://yew.rs/)

**Comment by arctic-hen7** [comment]
What exactly do you mean by this? If you mean Wasm workers, then only theoretically. You could use them, but there are no examples yet. However, this is most certainly on the roadmap.

Does Yew have built in support for Wasm workers? I had thought it just relied on `wasm-bindgen`, which Perseus does too. If that's the case, then all we're missing over here is some examples, which I'm happy to create or accept PRs for!

**Comment by lukechu10** [comment]
FYI yew now uses https://docs.rs/gloo-worker/latest/gloo_worker/ for web workers. It should be possible to use that API to spawn tasks on another thread (although there isn't any specific integration with perseus or sycamore yet).

**Reply by arctic-hen7** [reply]
Ah I see. Is there any use you could think of for that in the Perseus/Sycamore core code? It should be great for users (I'll add an example), but I don't think there's anything to explicitly change in the core...
